src/types/from_register_input.rs

```diff
@@ -36,15 +114,7 @@ impl TryFrom<RtpInputStream> for pipeline::RegisterInputOptions {
                     },
                 },
             }),
-            audio: audio.as_ref().map(|audio| input::rtp::InputAudioStream {
-                options: match audio {
-                    InputRtpAudioOptions::Opus {
-                        forward_error_correction,
-                    } => decoder::AudioDecoderOptions::Opus(decoder::OpusDecoderOptions {
-                        forward_error_correction: forward_error_correction.unwrap_or(false),
-                    }),
-                },
-            }),
+            audio,
```

Comment:
```suggestion
            audio: audio.map(TryInto::try_into).transpose()?,
```
Not tested, but I think sth like that should work